rest/src/main/groovy/whelk/rest/api/SearchUtils2.java

```diff
@@ -333,4 +328,52 @@ static class Offsets {
             }
         }
     }
+
+    static class Sort {
+        private enum Order {
+            asc,
```

Comment:
Should be capital letters?

Reply:
Made them lowercase to be compatible with ES DSL...
I think I will refactor and move all the ES stuff.